src/actions/assetsActions.js

```diff
@@ -630,14 +642,44 @@ export const resetSearchAssetsResultAction = () => ({
   type: RESET_ASSETS_SEARCH_RESULT,
 });
 
-export const checkForMissedAssetsAction = (transactionNotifications: Object[]) => {
+const getSupportedTokensAction = (supportedAssets: Asset[], currentAssets: AssetsByAccount, account: Account) => {
+  const accountId = getAccountAddress(account);
```

Comment:
```suggestion
  const accountId = getActiveAccountId(account);
```

Reply:
or
```suggestion
  const accountAddress = getAccountAddress(account);
```

Reply:
Cant use `getActiveAccountId` as it is used in the loop for all accounts